api/comments.rb

```diff
@@ -1,3 +1,32 @@
+get "#{APIPREFIX}/comments" do
+  query = Comment.all
+
+  # handle filters
+  error 400, "user_id is required" unless params['user_id']
+  query = query.where(author_id: params['user_id'])
+
+  error 400, "course_id is required" unless params['course_id']
+  query = query.where(course_id: params['course_id'])
+
+  if params["flagged"] == 'True'
+    query = query.where(:abuse_flaggers.nin => [nil, []])
+  end
```

Comment:
I think you should use `value_to_boolean` here instead of checking explictly if it's equal to `True`.